api/services/elastigroup/azure/spotVms/paths/groupVm.yaml

```diff
@@ -7,6 +7,14 @@ put:
     - Elastigroup Azure Spot VMs
   parameters:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
+    - in: "path"
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master